rpc/tests/test_validator_endpoints.py

```diff
@@ -1,4 +1,4 @@
-from rpc.tests.base import payload, post_request
+from common.testing.requests import payload, post_request
```

Comment:
@sevenearths aren't these e2e endpoints? they should be with the rest of the e2e tests, right?

Reply:
Right again. I'll move them